src/Dscp/Crypto/Encrypt.hs

```diff
@@ -96,7 +103,13 @@ mkPassPhrase bs
 data Encrypted ba = Encrypted
     { eAuthTag    :: !AuthTag
     , eCiphertext :: !ba
-    } deriving (Eq)
+    } deriving (Eq, Functor)
+
+instance Buildable ba => Show (Encrypted ba) where
+    show = toString . pretty
+
+instance Buildable ba => Buildable (Encrypted ba) where
+    build Encrypted{..} = "encrypted "+|eCiphertext|+""
```

Comment:
Why not simply use `base64` representation of serialised `Encrypted` (as I suggested to do also with Aeson instance?)

Reply:
Really, it's safe to show encrypted stuff, will do